pytest-itde/README.rst

```diff
@@ -1,5 +1,7 @@
 pytest-exasol-itde Plugin
 =========================
+This project has been discontinued. For integration tests running on SaaS please use the `pytest-backend` plugin.
```

Comment:
Please, use the full name exasol-pytest-backend and link to the pypi page of it

Reply:
OK, also it should be ITDE, not SaaS.